hazelcast/src/main/java/com/hazelcast/spi/impl/operationservice/impl/InvocationRegistry.java

```diff
@@ -65,21 +58,16 @@
  */
 public class InvocationRegistry {
 
-    private static final long SCHEDULE_DELAY = 1111;
     private static final int INITIAL_CAPACITY = 1000;
     private static final float LOAD_FACTOR = 0.75f;
-    private static final int DELAY_MILLIS = 1000;
     private static final double HUNDRED_PERCENT = 100d;
 
-    private final long backupTimeoutMillis;
-
     @Probe(name = "invocations.pending", level = MANDATORY)
     private final ConcurrentMap<Long, Invocation> invocations;
+    //todo: this dependency on nodeEngineImpl sucks.
```

Comment:
Do we need this? You could add this to (almost) every single field pointing to `NodeEngineImpl`.